commit 0495c3f18170f9690d32b5e0771232e4b92eb34f
Author: Radek Novacek <rnovacek@redhat.com>
Date:   Mon Aug 26 14:59:25 2013 +0200

    Set format(printf) attribute to __KReturn2 function
    
    This way, users of this function will get notified when they use options
    (like invalid number of formatted variables).

diff --git a/src/konkret/konkret.h b/src/konkret/konkret.h
index a9bb57d..40d66b3 100644
--- a/src/konkret/konkret.h
+++ b/src/konkret/konkret.h
@@ -205,8 +205,9 @@ KINLINE void KPutStatus(CMPIStatus* st)
         fprintf(stderr, "CMPIStatus{%u, %s}\n", st->rc, KChars(st->msg));
 }
 
+__attribute__((format(printf, 3, 4)))
 KINLINE CMPIStatus __KReturn2(
-    const CMPIBroker* cb, 
+    const CMPIBroker* cb,
     CMPIrc rc,
     const char* format,
     ...)
